tutorials/R/api-token/api-token.Rmd

```diff
@@ -0,0 +1,266 @@
+---
+syncID: 99d22b526c1a4a88847259a4d05b46af
+title: "Using an API Token when Accessing NEON Data"
+description: "Get an API token tied to your NEON user account, and use it for faster download speeds when accessing NEON data via the neonUtilities package."
+dateCreated: 2020-05-04
+authors: [Claire K. Lunch]
+contributors:
+estimatedTime: 20 minutes
+packagesLibraries: neonUtilities
+topics: data-management, rep-sci
+languageTool: R
+code1: R/api-token/api-token.R
+tutorialSeries:
+urlTitle: api-token
+
+---
+
+NEON data can be downloaded from either the NEON Data Portal or the NEON API. 
+When downloading from the Data Portal, you can create a user account. Read 
+about the benefits of an account on the <a href="https://www.neonscience.org/data/about-data/data-portal-user-accounts" target="_blank">User Account page</a>.
+
+Using a token is optional! You can download data without a token, and 
+without a user account. Using a token when downloading data via the API links 
+your downloads to your user account, as well as enabling faster download 
+speeds. As NEON Data Portal development continues, additional benefits will 
+be added to accounts and tokens, such as allowing you to access your own 
+download history, and to be notified of updates to data products you've 
+downloaded.
+
+For now, in addition to faster downloads, using a token helps NEON to track 
+data downloads. Using **anonymized** user information, we can then calculate 
+data access statistics, such as which data products are downloaded most 
+frequently, which data products are downloaded in groups by the same users, 
+and how many users in total are downloading data. This information helps NEON 
+to evaluate the growth and reach of the observatory, and to advocate for 
+training activities, workshops, and software development.
+
+Tokens can be used whenever you use the NEON API. In this tutorial, we'll 
+focus on using tokens with the neonUtilities R package.
+
+<div id="ds-objectives" markdown="1">
+
+## Objectives
+After completing this activity, you will be able to:
+
+ * Create a NEON API token 
+ * Use your token when downloading data with neonUtilities 
+
+## Things You’ll Need To Complete This Tutorial
+You will need the most current version of R and, preferably, `RStudio` loaded
+on your computer to complete this tutorial.
+
+### Install R Packages
+
+* **neonUtilities:** `install.packages("neonUtilities")`
```

Comment:
I'd add a note saying that this feature was added in version XXX as some individual may not think to update their neonUtilties package.